tests/contact-controller.test.js

```diff
@@ -60,6 +60,8 @@ test('Convert contacts to searchResults', async () => {
         categories: 'TBD',
         counselor: 'twilio-worker-id',
         notes: 'Lost young boy',
+        channel: '',
```

Comment:
This is insufficient testing.  It's good to test the default values but we should also test actual values as well.